packages/bitcore-node/src/modules/ethereum/types.ts

```diff
@@ -175,8 +177,8 @@ export interface EthTransactionJSON {
   nonce: number;
   to: string;
   from: string;
-  abiType?: IEthTransaction['abiType'];
-  decodedData?: AbiDecodedData;
+  abiType?: IAbiDecodedData;
+  decodedData?: IAbiDecodedData;
```

Comment:
Seems like we should rework this to only use either abiType or decodedData. That should probably be a separate PR though.

Reply:
Agreed.